modules/speech-to-text-workflowoperation/src/main/java/org/opencastproject/workflow/handler/speechtotext/SpeechToTextWorkflowOperationHandler.java

```diff
@@ -489,6 +487,11 @@ private String getMediaPackageLanguage(MediaPackage mediaPackage, WorkflowInstan
       language = StringUtils.trimToEmpty(mediaPackage.getLanguage());
     }
 
+    if (language.isEmpty()) {
+      // If nothing helps, use the fallback language
+      language = Objects.toString(operation.getConfiguration(LANGUAGE_FALLBACK), "en");
```

Comment:
We were never using language-fallback. Now we do.

Reply:
Commit 12bc12857d750851f65ab9e202f5cbd248306341 fixes the problem by removing the default, truly making this an optional argument.